coderd/workspaces.go

```diff
@@ -354,6 +354,15 @@ func (api *API) postWorkspacesByOrganization(rw http.ResponseWriter, r *http.Req
 		return
 	}
 
+	wsOwner, err := api.Database.GetUserByID(ctx, member.UserID)
+	if err != nil {
+		httpapi.Write(ctx, rw, http.StatusInternalServerError, codersdk.Response{
+			Message: "Requested workspace owner %q does not exist.",
+			Detail:  err.Error(),
+		})
+		return
+	}
+
```

Comment:
@mtojek does this change look OK to you?

Reply:
Do you suggest that exposing the avatar via `httpmw.OrganizationMemberParam` would be cleaner?

Reply:
I think it would be. UserAvatar is not protected data so it's the sort of thing we might just want to allow access to by default.

Reply:
Agree 👍